libcaf_core/caf/detail/profiled_send.hpp

```diff
@@ -31,31 +31,36 @@
 
 namespace caf::detail {
 
-template <class Self, class Sender, class Handle, class... Ts>
-void profiled_send(Self* self, Sender&& sender, const Handle& receiver,
+template <class Self, class SelfHandle, class Handle, class... Ts>
+void profiled_send(Self* self, SelfHandle&& src, const Handle& dst,
                    message_id msg_id, std::vector<strong_actor_ptr> stages,
                    execution_unit* context, Ts&&... xs) {
   CAF_IGNORE_UNUSED(self);
```

Comment:
Dont we use `[[maybe_unused]]` to silence unused warnings?

```suggestion
void profiled_send([[maybe_unused]] Self* self, SelfHandle&& src, const Handle& dst,
                   message_id msg_id, std::vector<strong_actor_ptr> stages,
                   execution_unit* context, Ts&&... xs) {
```